packages/slider/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "pcln-slider",
   "version": "2.0.0-beta.5",
   "description": "React component for range sliders",
-  "main": "dist/index.js",
+  "main": "dist/index.esm.js",
```

Comment:
As a good practice we should keep the `main` flag for `cjs` and the `module` flag for `esm` module. 
```
  "module": "dist/index.esm.js",  // for ES modules
  "main": "dist/index.cjs.js",  // for CommonJS modules
```